crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -284,10 +291,24 @@ async fn main() -> Result<()> {
         // Make the client runtime wait until we are done creating genesis objects
         let cloned_config = configs;
         let cloned_gas = primary_gas;
-        let auth_clients = GatewayState::make_authority_clients(
-            &gateway_config,
-            NetworkAuthorityClientMetrics::new_for_tests(),
+        let auth_clients = make_authority_clients(
+            &gateway_config.validator_set,
+            gateway_config.send_timeout,
+            gateway_config.recv_timeout,
+            network_authority_client_metrics.clone(),
         );
+
+        let committee = make_committee(gateway_config.epoch, &gateway_config.validator_set)?;
+        let committee_store = Arc::new(CommitteeStore::new_for_testing(&committee));
+        let aggregator = Arc::new(AuthorityAggregator::new(
+            committee,
+            committee_store,
+            auth_clients.clone(),
+            AuthAggMetrics::new(&registry),
+            Arc::new(SafeClientMetrics::new(&registry)),
+            Arc::new(NetworkAuthorityClientMetrics::new(&registry)),
+        ));
```

Comment:
If its not too difficult, could you factor these changes out so that they can be used by https://github.com/MystenLabs/sui/blob/44436ed30a6d7533c42602b4a808505eeec99c21/crates/sui-benchmark/tests/simtest.rs#L75 as well?

Reply:
will take a look

Reply:
@mystenmark `fn test_authority_aggregator` is used in multiple places, I will look to refactor all of them in a separate PR

Reply:
https://github.com/MystenLabs/sui/pull/5233